iceoryx_posh/source/roudi/roudi.cpp

```diff
@@ -230,7 +230,7 @@ void RouDi::processMessage(const runtime::IpcMessage& message,
             LogError() << "Wrong number of parameters for \"IpcMessageType::CREATE_CONDITION_VARIABLE\" from \""
                        << processName << "\"received!";
             errorHandler(
-                Error::kPORT_MANAGER__INTROSPECTION_MEMORY_MANAGER_UNAVAILABLE, nullptr, iox::ErrorLevel::MODERATE);
+                Error::kPORT_MANAGER__UNABLE_TO_ACQUIRE_CONDITION_VARIABLE, nullptr, iox::ErrorLevel::MODERATE);
```

Comment:
Please remove this entirely. It's not the fault of RouDi that an application provided the wrong number of parameters. It's an external event and RouDi should just tell the application it did something wrong.

Reply:
done